src/lua/compat.lua

```diff
@@ -51,6 +51,13 @@ initialization or new session creation.
 https://tarantool.io/compat/sql_seq_scan_default
 ]]
 
+local BOX_TUPLE_NEW_ARG_LIST_BRIEF = [[
+Whether `box.tuple.new` should interpret the argument list as an array of tuple
+field — this does not allow passing a tuple format as a second argument.
+
+https://tarantool.io/compat/box_tuple_new_arg_list_brief
```

Comment:
1. The `_brief` suffix in the URL looks redundant.
2. The page does not exist (you can look over the tarantool/doc repository or consult with @veod32 on this). At least, please, open a draft PR.

Reply:
1. Typo, removed it.
2. Added a documentation request to the corresponding commit.